php/utils/stack-function.php

```diff
@@ -408,14 +407,15 @@ public static function pre_pref( $apt_packages ) {
 			EE::info("Adding GPG key for NGINX");
 			EE_Repo::add_key( '3050AC3CD2AE6F03' );
 		}
-//
-//		if (in_array(EE_Variables::get_php_packages("7.0"), $apt_packages) || in_array(EE_Variables::get_php_packages('php5.6'), $apt_packages)) {
-//				EE::debug("Adding repository for PHP, please wait...");
-//			    EE_Repo::add(EE_Variables::get_php_repo());
-//			if ('debian' == EE_OS::ee_platform_distro()){
-//				EE_Repo::add_key('89DF5277');
-//				}
-//		}
+
```

Comment:
If this part of the code is unnecessary and will never be used, it should be removed, instead of commenting it out.